interface/modules/custom_modules/oe-module-claimrev-connect/openemr.bootstrap.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Modules\ClaimRevConnector;
```

Comment:
I don't think above line is needed (would confirm this with either @adunsulag or @sjpadgett )

Reply:
Its needed, or he needs more use <classname> statements.  Namespacing it like he's doing allows him to just grab classes that are in the same namespace.